<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>904</width>
    <height>728</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>630</y>
      <width>115</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>PushButton</string>
    </property>
   </widget>
   <widget class="QTabWidget" name="tabs">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>881</width>
      <height>541</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="policy">
     <attribute name="title">
      <string>Tab 1</string>
     </attribute>
     <widget class="QTableView" name="allPolicyTableView">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>170</y>
        <width>851</width>
        <height>341</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="dep">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>20</y>
        <width>71</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="arr">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>60</y>
        <width>71</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="officeNo">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>100</y>
        <width>71</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_4">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>20</y>
        <width>111</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="policyNo">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>130</y>
        <width>71</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>20</y>
        <width>91</width>
        <height>19</height>
       </rect>
      </property>
      <property name="text">
       <string>始发(三字码):</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>60</y>
        <width>81</width>
        <height>19</height>
       </rect>
      </property>
      <property name="text">
       <string>到达(三字码):</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>130</y>
        <width>62</width>
        <height>19</height>
       </rect>
      </property>
      <property name="text">
       <string>政策代码:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>100</y>
        <width>62</width>
        <height>19</height>
       </rect>
      </property>
      <property name="text">
       <string>Office号:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>20</y>
        <width>91</width>
        <height>19</height>
       </rect>
      </property>
      <property name="text">
       <string>最晚提前出票:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>60</y>
        <width>62</width>
        <height>19</height>
       </rect>
      </property>
      <property name="text">
       <string>票务说明:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_6">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>60</y>
        <width>113</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBox">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>130</y>
        <width>86</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>PAT</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="pnrCheck">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>100</y>
        <width>86</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>pnr</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBox_3">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>100</y>
        <width>86</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>行程单</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBox_4">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>130</y>
        <width>86</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>支付</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBox_5">
      <property name="geometry">
       <rect>
        <x>390</x>
        <y>100</y>
        <width>91</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>FY舱改备注</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="rateGt">
      <property name="geometry">
       <rect>
        <x>450</x>
        <y>130</y>
        <width>31</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QCheckBox" name="rateGtCheck">
      <property name="geometry">
       <rect>
        <x>390</x>
        <y>130</y>
        <width>61</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>返点&gt;</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="rateLtCheck">
      <property name="geometry">
       <rect>
        <x>510</x>
        <y>130</y>
        <width>61</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>返点&lt;</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="rateLt">
      <property name="geometry">
       <rect>
        <x>570</x>
        <y>130</y>
        <width>31</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>480</x>
        <y>20</y>
        <width>41</width>
        <height>19</height>
       </rect>
      </property>
      <property name="text">
       <string>留钱:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_9">
      <property name="geometry">
       <rect>
        <x>550</x>
        <y>20</y>
        <width>41</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="queryButton">
      <property name="geometry">
       <rect>
        <x>670</x>
        <y>20</y>
        <width>141</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>查询</string>
      </property>
     </widget>
     <widget class="QPushButton" name="dumpButton">
      <property name="geometry">
       <rect>
        <x>670</x>
        <y>60</y>
        <width>141</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>导出保存</string>
      </property>
     </widget>
     <widget class="QPushButton" name="dumpUpload">
      <property name="geometry">
       <rect>
        <x>670</x>
        <y>100</y>
        <width>141</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>导出保存并自动上传</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>480</x>
        <y>60</y>
        <width>81</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>航司(两字码):</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="companyCode">
      <property name="geometry">
       <rect>
        <x>570</x>
        <y>60</y>
        <width>41</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Tab 2</string>
     </attribute>
     <widget class="QTableView" name="tableView_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>60</y>
        <width>691</width>
        <height>301</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QProgressBar" name="progressBar">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>590</y>
      <width>801</width>
      <height>23</height>
     </rect>
    </property>
    <property name="value">
     <number>24</number>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>620</y>
      <width>801</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>904</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
